assets/src/scss/layout/_forms.scss

```diff
@@ -1,87 +1,6 @@
 .custom-control {
-  input[type="checkbox"] {
-    position: absolute;
-    opacity: 0;
-    pointer-events: none;
-
-    &:focus:focus-visible ~ label:before,
-    &:focus:focus-visible ~ .custom-control-description:before {
-      outline: 2px solid #99c3ff;
-      outline-offset: 1px;
-    }
-
-    &:not(:disabled) ~ .custom-control-description:hover:before,
-    &:not(:disabled) ~ label:hover:before {
-      border-color: var(--grey-900);
-    }
-
-    & ~ .custom-control-description,
-    & ~ label {
-      --control-label-- {
-        font-size: $font-size-xxxs;
-        line-height: 1rem;
-      }
-      font-family: var(--font-family-tertiary);
-      position: relative;
-      cursor: pointer;
-      display: inline-block;
-      padding-inline-start: $sp-4x;
-
-      a {
-        color: inherit;
-        font-weight: bold;
-      }
-
-      &:before {
-        content: "";
-        width: 20px;
-        height: 20px;
-        background: var(--white);
-        border: 1px solid var(--grey-500);
-        border-radius: 2px;
-        left: 0;
-        position: absolute;
-        top: calc((100% - $sp-4x) / 2);
-
-        html[dir="rtl"] & {
```

Comment:
Not neccesary to define a rule `html dir..` if you set `margin-inline-*` like here https://github.com/greenpeace/planet4-master-theme/pull/2441/files#diff-cdb1e0b8794b1c701944bd9d8e648d5915f3ca4aac5b664e823044fbd0da91beR376